ols/src/llms/providers/provider.py

```diff
@@ -98,18 +98,25 @@ class ProviderParameter:
 }
 
 # Generic to Azure OpenAI parameters mapping
-AzureOpenAIParametersMapping: dict[str, str] = {}
+AzureOpenAIParametersMapping: dict[str, str] = {
+    GenericLLMParameters.MAX_TOKENS_FOR_RESPONSE: "max_tokens",
+}
 
 # Generic to OpenAI parameters mapping
-OpenAIParametersMapping: dict[str, str] = {}
+OpenAIParametersMapping: dict[str, str] = {
+    GenericLLMParameters.MAX_TOKENS_FOR_RESPONSE: "max_tokens",
+}
 
 # Generic to BAM parameters mapping
-BAMParametersMapping: dict[str, str] = {}
+BAMParametersMapping: dict[str, str] = {
+    GenericLLMParameters.MIN_TOKENS_FOR_RESPONSE: "min_new_tokens",
```

Comment:
It is set in bam.py `"min_new_tokens": 1,`. The `MIN_TOKENS_FOR_RESPONSE` is valid parameter for BAM so I make sense to have it there. Otherwise you want to use it, you need to combine our parameter with original like
```python
# model config
min_new_tokens = X
max_tokens_for_response = Z
```
instead of 
```python
# model config
min_tokens_for_response = X
max_tokens_for_response = Z
```